src/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import { Link } from 'react-router-dom';
+import Styles from './styles/Header.module.css';
+
+const headerStyle = {
+  display: 'flex',
+  'margin-top': '0',
+  color: 'white',
+};
+const Header = () => (
+  <>
+    <nav style={headerStyle} className={Styles.myHeader}>
+      <h2>Math Magician</h2>
+      <ul>
+        <li>
+          <Link to="/" style={{ color: 'white' }}>Home</Link>
+        </li>
+        <li>
+          <Link to="/calculator" style={{ color: 'white' }}>Calculator</Link>
+        </li>
+        <li>
+          <Link to="/quotes" style={{ color: 'white' }}>Quotes</Link>
+        </li>
+      </ul>
+    </nav>
+  </>
+);
+export default Header;
```

Comment:
- [ ] Good job, but it would be nice if you remove the flex style you added on the `body` element so that this navbar can stay at the top, and not jump and shrink at any navigation :+1:

Reply:
@VICTOR-CHUKWU , thank you for the kind feedback, will definitely work on that.